tools/es5to6.js

```diff
@@ -59,7 +59,10 @@ git
         const unique = `${Date.now()}`.slice(-4);
         const es5Module = path.join(legacyPath, moduleId);
         const es6Module = path.join('static', 'src', 'javascripts', moduleId);
-        const es6Name = moduleId.split(path.sep).join('_').replace('.js', '');
+        const es6Name = moduleId
+            .split(path.sep)
+            .join('_')
+            .replace('.js', '');
```

Comment:
How interesting that `prettier` insists on splitting up this line, but not the line above which is of equal length 🤔

Reply:
> Print 3 or more chained calls on multiple lines